packages/core/src/components/TextArea/TextArea.tsx

```diff
@@ -76,6 +77,7 @@ const TextArea = forwardRef(
           aria-describedby={helpTextId ?? undefined}
           placeholder={placeholder}
           maxLength={maxLength}
+          {...rest}
```

Comment:
I usually like to set default behavior and then allow the end users to override it if they explicitly send the same props.
In this case, I agree it is better to move the `...rest` up because there are "aliases" to some of the props. For example `aria-invalid={error}` and  `rows={numRows}` (instead of `rows={rows}`

Moved it up 👍